cronjobs/nb-culler/cronjob.yaml

```diff
@@ -62,43 +62,42 @@ spec:
                           user_in_group2=true
                       fi
 
-                      if $user_in_group1 && $user_in_group2; then
-                          # Handle both group conditions
-                          if [[ $image != *$IMAGE_NAME_1* && $image != *$IMAGE_NAME_2* && $image != *$IMAGE_NAME_3* ]]; then
-                              echo "$nb is not using the correct images, deleting the notebook"
-                              oc delete notebook $nb -n $ns
-                              oc delete pvc $pvc -n $ns
-                          elif [[ $image == *$IMAGE_NAME_1* && $size != "X Small" ]]; then
-                              echo "$nb resource size is not correct for $IMAGE_NAME_1, deleting the notebook"
-                              oc delete notebook $nb -n $ns
-                              oc delete pvc $pvc -n $ns
-                          elif [[ ($image == *$IMAGE_NAME_2* || $image == *$IMAGE_NAME_3*) && $size != "Small" ]]; then
-                              echo "$nb resource size is not correct for $IMAGE_NAME_2 or $IMAGE_NAME_3, deleting the notebook"
-                              oc delete notebook $nb -n $ns
-                              oc delete pvc $pvc -n $ns
-                          elif [[ $image == *$IMAGE_NAME_1* && $difference -gt $CUTOFF_TIME_1 ]]; then
+                      # USER IS IN BOTH CLASSES
```

Comment:
I think user in both classes case if conditions could be simplified. You can first determine the larger cutoff_time and use that time to compare `$difference` and cutoff_time.